packages/appstore/src/components/wallet-header/__tests__/wallet-header.spec.tsx

```diff
@@ -1,27 +1,36 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import '@testing-library/jest-dom';
 import { mockStore, StoreProvider } from '@deriv/stores';
 import WalletHeader from '..';
-import { TAccountStatus } from 'Types';
-import { getStatusBadgeConfig } from '@deriv/account';
+import { TWalletAccount } from 'Types';
 
 const mockedRootStore = mockStore({});
-const setIsOpen = jest.fn();
 
 jest.mock('@deriv/account', () => ({
     ...jest.requireActual('@deriv/account'),
     getStatusBadgeConfig: jest.fn(() => ({ icon: '', text: '' })),
 }));
 
 describe('<WalletHeader />', () => {
+    let mocked_props: TWalletAccount;
+    beforeEach(() => {
+        mocked_props = {
+            is_demo: false,
+            currency: 'USD',
+            landing_company_name: 'svg',
```

Comment:
@thisyahlen-deriv Aren't `landing_company_name` and `landing_company_shortcode` the same? why do you pass 2 times?

Reply:
Because in the hook i have put it 2 times, one is deprecated, will remove it once everyone uses `landing_company_name` instead

Reply:
@thisyahlen-deriv you can use `landing_company_name` in your hook as well and don't wait for refactoring

Reply:
Will do 🙏